docs/README.md

```diff
@@ -0,0 +1,36 @@
+# Geoclimate user documentation
+
+
+
+**This documentation is under construction.**
+
+
+
+## The Geoclimate chain
+
+
+
+Geoclimate chain is a collection of spatial processes to produce vector maps of cities
+
+
+
+## Inputs
+
+The Geoclimate chain is currently working with two kind of dataset : OpenStreetMap, on a worldwide context and BD Topo (from [IGN](http://ign.fr/)), only on a the french scale.
+
+### OpenStreetMap
+
+
+
+### BD Topo (only in France)
+
+
+
+## Outputs
+
+As a result to the Geoclimate chain, 4 layers are produced :
```

Comment:
It depends on the chain used. 
There is a lot of missing information in this readme. I think we have to wait until it is more complete to validate it.

Reply:
That's why I wrote at the beginning "This documentation is under construction." ;-).
Up to you. I can remove the file if you prefer

Reply:
This file will be pushed in a separated PR. So I will remove it for this PR

Reply:
Sounds good

Reply:
done